pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -101,6 +105,46 @@ type CiliumBGPNeighborGracefulRestart struct {
 	RestartTimeSeconds *int32 `json:"restartTimeSeconds,omitempty"`
 }
 
+// BGPCommunityString holds a string containing a BGP Community Attribute value (RFC 1997)
+// as a 4-byte decimal number or two 2-byte decimal numbers separated by a colon,
+// or a BGP Large Community Attribute value (RFC 8092) as three 4-byte decimal numbers separated by a colon.
+// +kubebuilder:validation:Pattern=`^\d+(:\d+)?(:\d+)?$`
+type BGPCommunityString string
+
+// CiliumBGPPathAttributes can be used to apply additional path attributes
+// to matched routes when advertising them to a BGP peer.
+type CiliumBGPPathAttributes struct {
+	// SelectorType defines type of the object on which the Selector applies.
+	// - For "PodCIDR" the Selector matches k8s CiliumNode resources (resulting into PodCIDR routes).
```

Comment:
Not all k8s CiliumNode resources but just for the ones that this CiliumBGPPeeringPolicy applies to, correct? This is for the IPAM pod CIDRs, correct?

Reply:
Correct, this config only affects routes that are already being advertised based on `CiliumBGPPeeringPolicy`, does not cause advertising any new routes. Clarified it a bit more in the description.